testsuite/tests/lib-unix/win-symlink/parallel_symlink.ml

```diff
@@ -0,0 +1,39 @@
+(* TEST
+* libwin32unix
+include unix
+** bytecode
+** native
```

Comment:
```suggestion
** has_symlink
*** bytecode
*** native
```

(this takes care of whether symbolic links can be created and skips the test otherwise)